specification/cognitiveservices/data-plane/VisualSearch/v1.0/examples/SuccessfulVisualSearchRequest.json

```diff
@@ -0,0 +1,2408 @@
+                                                "availableSizesCount":1
+                                            },
+                                            "imageId":"B07F88FF5EBECF3E276445219755AA2719D975AB",
+                                            "accentColor":"B0921B"
+                                        },
+                                        {
+                                            "webSearchUrl":"https:\/\/www.bing.com\/images\/search?view=detailv2&FORM=OIIRPO&q=space+needle&id=FE52CFD00B7A8761700345AC0806EC23B1B256AD&simid=607993596948645705",
+                                            "webSearchUrlPingSuffix":"DevEx,5202.1",
+                                            "name":"Terumo Agani Needle - wowkeyword.com",
+                                            "thumbnailUrl":"https:\/\/tse2.mm.bing.net\/th?id=OIP.f7Fn32yMgO4VnDeXl0yYPQHaFj&pid=Api",
+                                            "datePublished":"2018-03-08T19:45:00.0000000Z",
+                                            "contentUrl":"http:\/\/www.budgetdumpster.com\/blog\/wp-content\/uploads\/2015\/02\/Space-Needle-1024x768.jpg",
+                                            "contentUrlPingSuffix":"DevEx,5204.1",
+                                            "hostPageUrl":"http:\/\/www.wowkeyword.com\/terumo-agani-needle\/",
+                                            "hostPageUrlPingSuffix":"DevEx,5203.1",
+                                            "contentSize":"153064 B",
+                                            "encodingFormat":"jpeg",
+                                            "hostPageDisplayUrl":"www.wowkeyword.com\/terumo-agani-needle",
+                                            "width":1024,
+                                            "height":768,
+                                            "thumbnail":{
+                                                "width":474,
+                                                "height":355
+                                            },
+                                            "imageInsightsToken":"ccid_f7Fn32yM*mid_FE52CFD00B7A8761700345AC0806EC23B1B256AD*simid_607993596948645705*thid_OIP.f7Fn32yMgO4VnDeXl0yYPQHaFj",
+                                            "insightsMetadata":{
+                                                "recipeSourcesCount":0,
+                                                "pagesIncludingCount":8,
+                                                "availableSizesCount":3
+                                            },
+                                            "imageId":"FE52CFD00B7A8761700345AC0806EC23B1B256AD",
+                                            "accentColor":"956236"
+                                        },
+                                        {
+                                            "webSearchUrl":"https:\/\/www.bing.com\/images\/search?view=detailv2&FORM=OIIRPO
```

Comment:
`tags[*]._type` is expected, according to the swagger.

Reply:
updated